addon/components/o-s-s/currency-input.hbs

```diff
@@ -1,16 +1,28 @@
 <div class="currency-input-container fx-1" ...attributes>
-  <div class="currency-input upf-input fx-row fx-1 fx-xalign-center">
-    <div class="currency-selector fx-row" role="button" {{on "click" this.toggleCurrencySelector}}>
-      <span class="margin-right-xxx-sm text-center symbol">{{this.selectedCurrencySymbol}}</span>
+  <div class="{{if this.onlyCurrencyInput "currency-input-onlycurrency" "currency-input"}} upf-input fx-row fx-1 fx-xalign-center ">
+    <div class="currency-selector fx-row fx-malign-space-between" role="button" {{on "click" this.toggleCurrencySelector}}>
+      <div class="fx-col">
+        <div class="fx-row fx-gap-px-24">
+          <span class="text-center symbol">{{this.selectedCurrencySymbol}}</span>
+          {{#if this.onlyCurrencyInput}}
```

Comment:
just use the `@currencyOnly` named arg and remove this getter (cf. https://github.com/upfluence/oss-components/pull/191/files#r915987263)